src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,165 @@
 package nextstep.subway.line;
 
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
+import nextstep.subway.line.dto.LineResponse;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 @DisplayName("지하철 노선 관련 기능")
 public class LineAcceptanceTest extends AcceptanceTest {
-    @DisplayName("지하철 노선을 생성한다.")
-    @Test
-    void createLine() {
-        // when
-        // 지하철_노선_생성_요청
-
-        // then
-        // 지하철_노선_생성됨
-    }
-
-    @DisplayName("기존에 존재하는 지하철 노선 이름으로 지하철 노선을 생성한다.")
-    @Test
-    void createLine2() {
-        // given
-        // 지하철_노선_등록되어_있음
-
-        // when
-        // 지하철_노선_생성_요청
-
-        // then
-        // 지하철_노선_생성_실패됨
-    }
-
-    @DisplayName("지하철 노선 목록을 조회한다.")
-    @Test
-    void getLines() {
-        // given
-        // 지하철_노선_등록되어_있음
-        // 지하철_노선_등록되어_있음
-
-        // when
-        // 지하철_노선_목록_조회_요청
-
-        // then
-        // 지하철_노선_목록_응답됨
-        // 지하철_노선_목록_포함됨
-    }
-
-    @DisplayName("지하철 노선을 조회한다.")
-    @Test
-    void getLine() {
-        // given
-        // 지하철_노선_등록되어_있음
-
-        // when
-        // 지하철_노선_조회_요청
-
-        // then
-        // 지하철_노선_응답됨
-    }
-
-    @DisplayName("지하철 노선을 수정한다.")
-    @Test
-    void updateLine() {
-        // given
-        // 지하철_노선_등록되어_있음
-
-        // when
-        // 지하철_노선_수정_요청
-
-        // then
-        // 지하철_노선_수정됨
-    }
-
-    @DisplayName("지하철 노선을 제거한다.")
-    @Test
-    void deleteLine() {
-        // given
-        // 지하철_노선_등록되어_있음
-
-        // when
-        // 지하철_노선_제거_요청
-
-        // then
-        // 지하철_노선_삭제됨
-    }
+	Map<String, String> params;
+
+	@BeforeEach
+	void setBeforeEach() {
+		params = new HashMap<>();
+		params.put("color", "br-red-600");
+		params.put("name", "신분당선");
+	}
+
+	@DisplayName("지하철 노선을 생성한다.")
+	@Test
+	void createLine() {
+		// when
+		ExtractableResponse<Response> response = createLine(params);
+
+		// then
+		assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+		assertThat(response.header("Location")).isNotBlank();
+	}
+
+	@DisplayName("기존에 존재하는 지하철 노선 이름으로 지하철 노선을 생성한다.")
+	@Test
+	void createLine2() {
```

Comment:
요구사항에 보시면 아래와 같은 내용이 있습니다 😄 
> 인수 테스트의 각 스텝들을 메서드로 분리하여 재사용하세요.
ex) 인수 테스트 요청 로직 중복 제거 등

테스트 코드 작성 잘 해주셨는데 이번 과정 만큼은 **인수테스트** 이므로
개발자(혹은 직군)가 아닌 다른 직군 (기획, 디자인, 사업 등) 구성원들이 함께 
이 코드를 보면서 테스트를 하거나 문제점을 알 수 있게 하는 취지로 연습해보시는 건 어떨까요?? 😄 

그렇다면 무엇보다도 가독성 개선이 우선되어야 하고, 
시나리오나 유저스토리가 잘 드러나도록 되어있어야 합니다 😄 
통상적으로는 한글명 `method`를 잘 사용하지 않지만 인수테스트이므로 
이 부분만 개선되면 더 좋을 것 같습니다 😄 



```suggestion
	void createLine2() {
        // given
        지하철_노선_등록되어_있음(LINE_NUMBER_9);

        // when
        ExtractableResponse<Response> response = 지하철_노선_생성_요청(LINE_NUMBER_9);

        // then
        지하철_노선_생성_실패됨(response);
    }
```